lib/active_merchant/billing/gateways/redsys.rb

```diff
@@ -226,6 +227,7 @@ def authorize(money, payment, options = {})
         data[:description] = options[:description]
         data[:store_in_vault] = options[:store]
         data[:sca_exemption] = options[:sca_exemption]
```

Comment:
Does this also need the new `Y` logic?

Reply:
Never mind - I realize that this is just setting up the data and not actually appending to the request.